src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -3633,11 +3633,11 @@ public static void Main()
     {
         if (null is 1) {}
         if (Main is 2) {}
-        if (delegate {} is 3) {}
+        if ((delegate {}) is 3) {}
```

Comment:
I think the best thing would be to leave the original source alone and paste in the new actual diagnostics to VerifyDiagnostics